pipeline/cleanup.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+
+echo "Clearing out resources of lambda-file-refarch and Pipeline stacks..."
```

Comment:
Do we want to clear out all the things, or just the resources created in this stack?

Reply:
This stack creates an application stack automatically, and you cannot remove the pipeline first because it uses a CFN role to deploy the application. We could inspect the pipeline stack for the AppName and remove that but this is assuming you have deployed the defaults. 

I will add a note to the readme.

Reply:
Thinking about it. I will just add -app to the stack name. That way it will isolate the pipeline version from the other one.